obs/libs/utils.py

```diff
@@ -63,5 +64,31 @@ def human_date(unixtime):
     return human_datetime
 
 
+def set_id(ids):
+    """Set ID  for granting access"""
+    if ids != None:
+        ids = [f"id={id}" for id in ids.split(",")]
+        return ",".join(ids)
+    else:
+        return ""
+
+
 def compatibility():
+    """get neo compatibility environment"""
     return strtobool(os.environ.get("OBS_USE_NEO"))
+
+
+def user_sanitize(args):
+    """Sanitize input when creating new user"""
+    arg = {}
```

Comment:
I thought `args`  has many  args parameter name, this function only for args with key `userId` .  You can define some general function name and pass based on args.

Reply:
done, change it to "uID"